src/EdFi.AnalyticsMiddleTier.DataStandard31/Base/MSSQL/0021-View-SectionDim-Create.sql

```diff
@@ -0,0 +1,27 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'SectionDim')
+BEGIN
+	DROP VIEW analytics.SectionDim
+END
+GO
+
+CREATE VIEW analytics.SectionDim AS
+    SELECT DISTINCT 
+          CAST(s.SchoolId AS VARCHAR) AS SchoolKey, 
+          CAST(s.SchoolId AS NVARCHAR) + '-' + s.LocalCourseCode + '-' + CAST(s.SchoolYear AS NVARCHAR) + '-' + s.SectionIdentifier + '-' + s.SessionName AS SectionKey, 
+		  CONCAT(s.LocalCourseCode,'-',SessionName) AS SectionName,
+		  SessionName,
+          LocalCourseCode, 
+          SchoolYear, 
+          EducationalEnvironmentDescriptorId AS EducationalEnvironmentTypeId, 
```

Comment:
Replace this column by Descriptor.Description. (Add a join with Descriptor -> EducationalEnvironmentDescriptorId=DescriptorId)

Reply:
updated